src/components/Form/FormSelect.react.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+type Props = {| +children?: React.Node, +className?: string |};
+
+function FormSelect({ className, ...props }: Props): React.Node {
+  const classes = cn(
+    { "form-control": true, "custom-select": true },
+    className
+  );
+  return <select className={classes} {...props} />;
```

Comment:
Will the `{...props}` render children here?  Even if it does, it may make it more readable to manually put children between the <select> and </select> tags.

Reply:
Yeah it will render them but I also agree on the readability aspect, will start doing it this way